sygnal/sygnal.py

```diff
@@ -89,19 +85,6 @@ def __init__(self, config, custom_reactor, tracer=opentracing.tracer):
                 )
                 config["proxy"] = proxy_url
 
-        # Old format db config
-        if config.get("db") is not None:
-            logger.warning(
-                "Config includes the legacy 'db' option, please migrate"
-                " to 'database' instead"
-            )
-            config["database"] = {
-                "name": "sqlite3",
-                "args": {"dbfile": config["db"]["dbfile"]},
-            }
```

Comment:
Perhaps instead of removing this, we might fancy telling people to remove db/database config fields? Is that sensible?

Reply:
Interesting! Yes, I think we could do that-the DB config fields should already be pre-removed, but I see no reason why we can't warn someone if for some strange reason they are not.